yb-voyager/src/tgtdb/target_db_interface.go

```diff
@@ -120,15 +118,25 @@ func (args *ImportBatchArgs) GetYBCopyStatement() string {
 	return fmt.Sprintf(`COPY %s %s FROM STDIN WITH (%s)`, args.TableName, columns, strings.Join(options, ", "))
 }
 
-func (args *ImportBatchArgs) GetSqlLdrControlFile(schema string) string {
+func (args *ImportBatchArgs) GetSqlLdrControlFile(schema string, tableSchema map[string]map[string]string) string {
 	var columns string
 	if len(args.Columns) > 0 {
 		columnsSlice := make([]string, 0, len(args.Columns))
 		for _, col := range args.Columns {
 			// Add the column name and the NULLIF clause after it
-			columnsSlice = append(columnsSlice, fmt.Sprintf(`%s NULLIF %s='\\N'`, col, col))
+			dataType := tableSchema[col]["__debezium.source.column.type"] 
```

Comment:
Add a TODO comment here as well. 
It seems mixed up for the importBatch layer to have awareness of debezium. Given that we absolutely need it - we should call it something else - like `source_db_datatype`